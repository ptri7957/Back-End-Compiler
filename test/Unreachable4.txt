( (unreachable (a)
    (0 (lc r1 2)
       (st x r1)
       (ld r2 x)
       (ret r2) )
    (1 (lc r3 80)
       (st y r3)
       (ld r4 y)
       (ret r4) ) ) )