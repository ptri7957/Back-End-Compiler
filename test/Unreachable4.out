( (unreachable (a)
    (0 (lc r1 2)
       (st x r1)
       (ld r2 x)
       (ret r2) ) ) )